### PR TITLE
fix disabled dropdown submenu style error

### PR DESCRIPTION
**Pull request by laispace**
Close: https://github.com/ant-design/ant-design/issues/6548

### CODE REVIEW

**Comment by afc163** [comment]
image

**Comment by laispace** [comment]
@afc163 已修正这个错误信息

File: components/dropdown/style/index.less [hunk 1]
```diff
@@ -49,7 +49,7 @@
     background-clip: padding-box;
 
     &-item,
-    &-submenu-title {
+    &-submenu {
```
Comment (benjycui) [inline components/dropdown/style/index.less]: Why did you remove this?

Comment (laispace) [inline components/dropdown/style/index.less]: I compared Menu's style and Dropdown's style, finding that this line fixed the problem, without side effects.

Comment (laispace) [inline components/dropdown/style/index.less]: If I am wrong, maybe you can refuse the PR, but I still want to know what's the correct way to fix the problem.

**Comment by benjycui** [comment]
Close in 7ff0573
